ibex_icache.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:ibex_icache:0.1"
```

Comment:
The IP cores in the ibex repository are in the `lowrisc:ibex` namespace.

Reply:
Thanks for the pointer, will update this accordingly.